frontend/tests/controllers/ProblemUpdateTest.php

```diff
@@ -1789,7 +1801,37 @@ public function testUpdateProblemSettingsAndVisibility() {
         $this->assertEquals(1, $response['visibility']);
     }
 
-    public function testUpdateProblemWithDifferentShowDiffValues() {
+    /**
+     * @dataProvider showDiffValueProvider
+     */
+    public function testUpdateProblemWithValidShowDiffValues(
+        string $showDiffValue
+    ) {
+        // Get a problem
+        $problemData = \OmegaUp\Test\Factories\Problem::createProblem();
+
+        // Call API
+        $login = self::login($problemData['author']);
+
+        $response = \OmegaUp\Controllers\Problem::apiUpdate(
```

Comment:
nit:

```suggestion
        \OmegaUp\Controllers\Problem::apiUpdate(
```

si no se usa, no hay que asignarle un nombre.

Reply:
Listo